openvdb/openvdb/unittest/main.cc

```diff
@@ -12,11 +12,24 @@
 #include <string>
 #include <vector>
 
+#if defined(__linux__)
+#include <fenv.h>
+#endif
+
 #include <gtest/gtest.h>
 
 int
 main(int argc, char *argv[])
 {
     ::testing::InitGoogleTest(&argc, argv);
+
+#if defined(__linux)
+    const char* s = getenv("VDB_TEST_FPE");
```

Comment:
This looks good to me in general, thanks for fixing these issues. However, instead of this being enabled with an environment variable, I would prefer that this be a CMake option such as `OPENVDB_TESTS_FPE` that we could more easily enable in our CI, somewhere like this:

https://github.com/AcademySoftwareFoundation/openvdb/blob/master/openvdb/openvdb/unittest/CMakeLists.txt#L15

I wouldn't expect that it is useful to be able to enable/disable FPEs at runtime.